centovacast.php

```diff
@@ -1792,7 +1794,10 @@ private function generatePassword($min_length = 5, $max_length = 10)
         for ($i = 0; $i < $length; $i++) {
             $password .= substr($pool, mt_rand(0, $pool_size - 1), 1);
         }
-
+		
+		//Make password easier to read
```

Comment:
That seems pretty subjective, but if you are going to do this you should just update the character pool to contain upper case characters instead of lower case ones.